docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md

```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
+
+_Note: This is only applicable if your Glue job was created using the Glue Studio UI._
+
+1. Navigate back to the new script file you created in the GitHub UI (or existing script file, if you are updating your script), and paste your script.
+1. Submit your changes by referring to the [Committing changes][committing-changes] section of the **Using Github** guide
+   The Data Platform team needs to approve any changes to the code, so your change won't happen automatically.
+
+:::important Updating your Glue job script
+If you need to update your Glue job script in the future, you must follow steps 4-6 to have those changes deployed to the Data Platform Project.
 :::
 
-- Open the [scripts directory][scripts-directory] in the Data Platform Project and navigate to your department folder
-- Click `Add file` and then `Create new file`
-- Name your file, using underscores as separators e.g. `address_cleaning_housing_repairs`, and then paste your code in field below
-- Follow [the instructions here][committing-changes] to commit your new file to the project
-
 ### 2. Copy an existing Glue job module block or use the example below and populate the necessary variables
 
-Once the Pull Request to add your script has been approved, and you have confirmed that your script is in the Data Platform Project,
-you can proceed with this step.
-In this step, you will be creating a module which will deploy your Glue job (and crawler) to the Data Platform Project.
+Once the Pull Request to add or update your script has been approved, and you have confirmed that your script is in the Data Platform Project,
+you can proceed to follow the steps in this section.
 
-- Open the [Data Platform Project][data-platform-project] in GitHub
-  If you don't have the correct permissions, you'll get a '404' error (see [prerequisites][prerequisites]).
-- Navigate to the main `terraform` directory. Open `23-aws-glue-jobs-<YOUR-DEPARTMENT-NAME>.tf`
-  - _if this file does not exist, create a new terraform file for your department by clicking `Add file` then `Create new file`.
-    The name of the new file should be:_
-    ```
-    23-aws-glue-jobs-<YOUR-DEPARTMENT-NAME>.tf
-    ```
-    _Refer to the [example](#example-module-block) below to get started._
-- Click `edit` or the pencil icon and copy and paste an existing module block.
-- When updating the module:
-  - Do not change any spacing before the `=`
-  - Do not remove the `"quotation marks"` around any values which use them
-  - When adding a variable that does not already exist, paste the variable name and then intended value after the `=`
-    and ensure that all equal signs (`=`) line up as shown in the example
+In this section, you will be creating a module (or updating an existing one if you need to make changes to an existing Glue job) which will deploy your Glue job (and crawler) to the Data Platform Project.
```

Comment:
A module won't mean anything to most people. Maybe reword to something like "you will be writing the code which will deploy..."